src/test/java/com/tenx/ms/retail/rest/TestClientController.java

```diff
@@ -0,0 +1,177 @@
+package com.tenx.ms.retail.rest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.tenx.ms.commons.config.Profiles;
+import com.tenx.ms.commons.rest.RestConstants;
+import com.tenx.ms.commons.rest.dto.ResourceCreated;
+import com.tenx.ms.commons.tests.AbstractIntegrationTest;
+import com.tenx.ms.retail.RetailServiceApp;
+import com.tenx.ms.retail.client.rest.dto.Client;
+import org.apache.commons.io.FileUtils;
+import org.flywaydb.test.annotation.FlywayTest;
+import org.flywaydb.test.junit.FlywayTestExecutionListener;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.TestRestTemplate;
+import org.springframework.boot.test.WebIntegrationTest;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by goropeza on 25/08/16.
+ */
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@WebIntegrationTest(randomPort = true)
```

Comment:
This should be inherited from the extended class (AbstractIntegrationTest) of this test class. There's no need to re-declare it here